sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -107,19 +79,18 @@ SUBROUTINE CLDRAD
                          HBOT, HBOTD, HBOTS, HTOP, HTOPD, HTOPS,  FIS, PBLH,  &
                          PBOT, PBOTL, PBOTM, PBOTH, CNVCFR, PTOP, PTOPL,      &
                          PTOPM, PTOPH, TTOPL, TTOPM, TTOPH, PBLCFR, CLDWORK,  &
-                         ASWIN, AUVBIN, AUVBINC, ASWOUT,ALWOUT, ASWTOA,       &
```

Comment:
@kayeekayee It seems you invert the changes in PR #453. Please remove duplicated ASWIN.